__tests__/events/subscribers.ts

```diff
@@ -112,3 +112,35 @@ describe("event subscriber removal", () => {
     await deleteUsers();
   });
 });
+
+describe("invalid subscriber removal", () => {
```

Comment:
seems like a wrong description here for the test involved

Reply:
let me make this clearer real quick